src/update.c

```diff
@@ -61,7 +61,7 @@ void update_jou_compiler()
         fail();
 #else
     confirm("Run \"git pull && make\"?");
```

Comment:
On NetBSD it would be nice if this said `Run "git pull && gmake"?`

Reply:
I don’t know, maybe?